source/html/js/app/ui/diagram_menu.js

```diff
@@ -228,19 +228,19 @@ $("#dupe_diagram_dialog").on("shown.bs.modal", function () {
 });
 
 $("#dupe_diagram_dialog_proceed").on("click", function () {
-    var current_diagram = diagrams.shown();
+    const current_diagram = diagrams.shown();
     if (current_diagram) {
         try {
             // get the name
-            var name = filterXSS($("#dupe_diagram_dialog_name").val());
+            const name = filterXSS($("#dupe_diagram_dialog_name").val());
             // check it
-            var valid_name = new RegExp("^\\w+");
+            const valid_name = /^\w+/;
             if (valid_name.test(name)) {
                 // create a new diagram
-                var new_diagram = diagrams.add(name, _.snakeCase(name), true);
+                const new_diagram = diagrams.add(name, _.snakeCase(name), true);
```

Comment:
should this be let because it changes after initiation?

Reply:
as long as the variable does not get re-assigned, const should be fine I think.